core/iwasm/libraries/wasi-nn/src/wasi_nn_tensorflowlite.cpp

```diff
@@ -206,6 +210,17 @@ tensorflowlite_init_execution_context(void *tflite_ctx, graph g,
                 NN_ERR_PRINTF("Error when enabling GPU delegate.");
                 use_default = true;
             }
+#elif defined(WASI_NN_ENABLE_EXTERNAL_DELEGATE)
+            NN_WARN_PRINTF("external delegation enabled.");
+            auto options =
+                TfLiteExternalDelegateOptionsDefault(WASI_NN_EXT_DELEGATE_PATH);
+            auto *delegate = TfLiteExternalDelegateCreate(&options);
```

Comment:
Had better check the return value? And same to do for L206?

Reply:
+1